javascript/ui.js

```diff
@@ -409,42 +409,54 @@ async function browseModels(){
 }
 
 // get user info
-window.onload = function () {
-    const hostOrigin = location.origin;
+window.addEventListener("load", (event) => {
```

Comment:
Do you want to change this to `onUiLoad` function, which will be called at the right timing, instead of using the `setTimeout` function?

Reply:
its already changed, the file is outdated